From e50bb712db7b72b9c83ebfb64646efd211e72104 Mon Sep 17 00:00:00 2001
From: Jerome Jutteau <jerome.jutteau@outscale.fr>
Date: Tue, 11 Apr 2023 16:46:25 +0200
Subject: [PATCH] Add rustls-tls feature to make static binaries

Signed-off-by: Jerome Jutteau <jerome.jutteau@outscale.fr>
---
 Cargo.toml | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 548c0c1..524b7ea 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -13,10 +13,14 @@ serde = "^1.0"
 serde_derive = "^1.0"
 serde_json = "^1.0"
 url = "^2.2"
-reqwest = "~0.9"
+reqwest = {version = "~0.9", default-features = false}
 aws-sigv4 = "0.3.0"
 http = "0.2.5"
 secrecy = "0.8.0"
 
 [dev-dependencies]
 rand = "0.8.5"
+
+[features]
+default = ["reqwest/default-tls"]
+rustls-tls = ["reqwest/rustls-tls"]
\ No newline at end of file
-- 
2.38.1

